label_sync/labels.yaml

```diff
@@ -353,6 +353,23 @@ repos:
         name: design-help-wanted
         target: issues
         addedBy: humans
+  tektoncd/operator:
+    labels:
+      - color: 160a72
+        description: This issue is used during working group to mark issues to be discussed
+        name: grooming
+        target: issues
+        addedBy: humans
+      - color: 0c6a89
+        description: This issue targets the OpenShift Platform
+        name: platform/openshift
+        target: issues
+        addedBy: humans
+      - color: 0c6a89
+        description: This issue is used during working group to mark issues to be discussed
```

Comment:
The description for this and `grooming` are the same - I'm assuming this one should be something else?